tests/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-inappbrowser-tests",
-  "version": "3.1.0",
+  "version": "3.1.0-OS",
```

Comment:
It is a fix in the sense that all the other files already had the correct version, that one was missing.

Regarding the time to raise the version, usually we do it after the PR is merged but these pull from upstream cases are weird, since the pulled commits affect the version lines. To raise the version later we need to either ignore the suffix or the upstream version. It's more of a question of "merging" the version than exactly raising it, therefore IMO it is fine doing it at this stage. I think the best approach is debatable but not particularly relevant.

Reply:
> I think the best approach is debatable but not particularly relevant.
I agree, hence the approaval ;)